packages/@react-aria/interactions/test/usePress.test.js

```diff
@@ -15,7 +15,8 @@ import React from 'react';
 import {usePress} from '../';
 
 function Example(props) {
-  let {elementType: ElementType = 'div', ...otherProps} = props;
+  // eslint-disable-next-line no-unused-vars
+  let {elementType: ElementType = 'div', onPress, onPressChange, onPressStart, onPressEnd, onPressUp, ...otherProps} = props;
```

Comment:
how do we actually want to pull this apart? filterdomprops?

Reply:
IMO, it feels like usePress should do some filtering inside of it since it is making the assumption that all props passed to it (barring onPress, onPressStart, etc) are DOM props. Maybe it should run filterDOMProps on domProps?

Prob out of scope of this PR, prob something we should discuss as a whole